src/renderers/dom/fiber/ReactDOMFiber.js

```diff
@@ -322,9 +324,15 @@ function renderSubtreeIntoContainer(parentComponent : ?ReactComponent<any, any,
     while (container.lastChild) {
       container.removeChild(container.lastChild);
     }
-    root = container._reactRootContainer = DOMRenderer.createContainer(container);
+    const newRoot = DOMRenderer.createContainer(container);
```

Comment:
To satisfy Flow, which complains otherwise. I'm guessing because it doesn't know that `syncUpdates` executes its callback synchronously.